src/export/StructureDefinitionExporter.ts

```diff
@@ -383,26 +384,16 @@ export class StructureDefinitionExporter implements Fishable {
         : 'constraint';
 
     if (fshDefinition instanceof Extension) {
-      // context and contextInvariant only apply to extensions.
-      // Keep context, assuming context is still valid for child extensions.
-      // Keep contextInvariant, assuming context is still valid for child extensions.
-
       // Automatically set url.fixedUri on Extensions unless it is already set
       const url = structDef.findElement('Extension.url');
       if (url.fixedUri == null) {
         url.fixedUri = structDef.url;
       }
+      // context and contextInvariant only apply to extensions.
+      // Keep context, assuming context is still valid for child extensions.
+      // Keep contextInvariant, assuming context is still valid for child extensions.
       if (structDef.context == null) {
-        // Set context to everything by default, but users can override w/ rules, e.g.
-        // ^context[0].type = #element
-        // ^context[0].expression = "Patient"
-        // TODO: Consider introducing metadata keywords for this
-        structDef.context = [
-          {
-            type: 'element',
-            expression: 'Element'
-          }
-        ];
+        this.setContext(structDef, fshDefinition);
       }
```

Comment:
I don't think we talked about what to do with inherited contexts... The way it is implemented now, if a parent defined a context, then even if the author specified a `Context:`, it would be dropped (since `this.structDef context === null` would be `false`). I don't think we want that.

I think that if there is an inherited context and the author specifies a `Context:`, it should _overwrite_ the existing context. The other option would be to append to the existing context, but I don't think this is right because a profiled extension might be more specific than its parent and therefore apply to fewer (or more specific) contexts as well.